draft-ietf-ace-edhoc-oscore-profile.md

```diff
@@ -259,81 +260,39 @@ An example of such a request is shown in {{token-request-update}}.
 ~~~~~~~~~~~~~~~~~~~~~~~
 {: #token-request-update title="Example of C-to-AS POST /token request for updating access rights to an access token."}
 
-## AS-to-C: Access Token Response # {#as-c}
+## AS-to-C: Response # {#as-c}
 
-After verifying the POST request to the /token endpoint and that C is authorized to obtain an access token corresponding to its access token request, AS responds as defined in {{Section 5.8.2 of RFC9200}}. If the request from C was invalid, or not authorized, AS returns an error response as described in {{Section 5.8.3 of RFC9200}}.
+After verifying the POST request to the /token endpoint and that C is authorized to access a resource following its request, AS responds as defined in {{Section 5.8.2 of RFC9200}}, with potential modification as detailed below. If the request from C was invalid, or not authorized, AS returns an error response as described in {{Section 5.8.3 of RFC9200}}.
```

Comment:
The access is requested for a scope applied to an audience. That can be more than a resource.

A middle ground between the new and old sentence can be:

"After verifying the POST request to the /token endpoint and that C is authorized to perform the access specified in its request, ..."

Reply:
The authorization may be different from what is specified in the request. Changed to:

"After verifying the POST request to the /token endpoint that C is authorized to access, ..."